utils/dc-chain/scripts/common.sh

```diff
@@ -114,8 +114,8 @@ export ARM_ISL_TARBALL_URL=gcc.gnu.org/pub/gcc/infrastructure/isl-${ARM_ISL_VER}
 # Retrieve the web downloader program available in this system.
 export IS_CURL=0
 export WEB_DOWNLOADER=
-curl_cmd=`get_make_var curl_cmd scripts/init.mk`
-wget_cmd=`get_make_var wget_cmd scripts/init.mk`
+curl_cmd="curl --fail --location -C - -O"
+wget_cmd="wget -c"
```

Comment:
Why was this changed?

Reply:
These were changed due to converting the variables in the Makefile to functions which now take parameters of the URL and optionally the output filename:
```
wget_cmd = wget -c $(if $(2),-O $(2)) '$(1)'
curl_cmd = curl --fail --location  -C - $(if $(2),-o $(2),-O) '$(1)' 
```

Directly pulling this out of the Makefile into the bash script like was done previously would no longer work so I just set them to the previous values that would be obtained from the Makefile to keep the script functioning as it was.